Changes to Calda's description

Naminson opened the pull request:
**Content (Artwork / Missions / Jobs)**

## Summary
This PR changes the description of Calda.
The description is mostly unchanged, but now the reason the carbon dioxide incident doesn't effect the economy is more due to the population forgetting about it, rather than the Paradise Planets being cartoonishly egotistical. This is a very small change, but that part of the description is something that's annoyed me ever since I started playing the game.
There's also a slight change in the wording of the spaceport to refer to "merchant captains" instead of "a captain such as yourself", since I don't know of any other instances it refers to you like that.

Amazinite commented on data/map.txt:
```diff
@@ -25440,9 +25440,9 @@ planet Calda
 	landscape land/lava4
 	description `Calda is a tropical resort world, a pleasant place to unwind after a few days skiing on its sister planet, Vail. The most popular tourist destination is the hot springs, and the spa complexes that have been built up around them. Constant pesticide use has nearly eliminated insect pests. Because of the uncomfortably warm and humid summer months, the very rich view Calda more as a tou ...
-	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population in moments. This incident has had hardly any effect on the tourism industry. Many of the regulars believe they are rich enough that no volcano would dare provoke them in such a way.`
+	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population. This incident shocked the Paradise Worlds and lead to a drop in tourism here for months.`
```
I'd extend the sentence slightly to mention that it actually is the case that people mostly forgot about it.
```suggestion
	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population. This incident shocked the Paradise Worlds and led to a drop in tourism here for months, but the industry quickly recovered after everyone had forgotten about it.`
```

Naminson commented:
I rewrote the bit about the tourists forgetting. I think I've addressed your problems. I also think it might run a little long, but I compared it to some other planets and it seems fine.

Amazinite commented on data/map.txt:
```diff
@@ -25440,9 +25440,9 @@ planet Calda
 	landscape land/lava4
 	description `Calda is a tropical resort world, a pleasant place to unwind after a few days skiing on its sister planet, Vail. The most popular tourist destination is the hot springs, and the spa complexes that have been built up around them. Constant pesticide use has nearly eliminated insect pests. Because of the uncomfortably warm and humid summer months, the very rich view Calda more as a tou ...
-	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population in moments. This incident has had hardly any effect on the tourism industry. Many of the regulars believe they are rich enough that no volcano would dare provoke them in such a way.`
+	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population. This incident shocked the Paradise Worlds and led to a very large drop in tourism here. Fortunetly for Calda's economy, this effect ceased within six months, which was about how long it took for most potiential tourists to forgot about the in ...
```
```suggestion
	description `	In Calda's early days as a spa planet, a nearly silent eruption of carbon dioxide filled a resort valley and suffocated the whole population. This incident shocked the Paradise Worlds and led to a very large drop in tourism here. Fortunately for Calda's economy, this effect ceased within six months, which was about how long it took for most potential tourists to forgot about the incident.`
```
